src/fides/api/service/privacy_request/request_runner_service.py

```diff
@@ -446,6 +449,19 @@ def run_privacy_request(
                 logger.info("Terminating privacy request: request deleted.")
                 return
 
+            duplicate_detection_service = DuplicateDetectionService(session)
+            # Service initializes with ConfigProxy, so we can check if duplicate detection is enabled
+            if duplicate_detection_service.config.enabled:
```

Comment:
Would it be better to just have a property `duplicate_detection_service.enabled`? I think the config should be a hidden implementation detail (i.e. if you wanted to test the service you can just stub out `enabled=True` which is simpler). Also, you could move this check into `is_duplicate_request` and if it's not enabled, just immediately return False